docs/schema/azure.md

```diff
@@ -845,3 +871,391 @@ Representation of an [AzureStorageBlobContainer](https://docs.microsoft.com/en-u
+
+### Relationships
+
+- Azure Subscription contains one or more database accounts.
+
+        ```
+        (AzureSubscription)-[RESOURCE]->(AzureCosmosDBAccount)
+        ```
+- Azure Database Account has write permissions from, read permissions from and is associated with Azure CosmosDB Locations.
+
+        ```
+        (AzureCosmosDBAccount)-[WRITE_PERMISSIONS_FROM]->(AzureCosmosDBLocation)
+        ```
+        (AzureCosmosDBAccount)-[READ_PERMISSIONS_FROM]->(AzureCosmosDBLocation)
+        ```
+        (AzureCosmosDBAccount)-[ASSOCIATED_WITH]->(AzureCosmosDBLocation)
+        ```
+- Azure Database Account contains one or more Cors Policy.
+
+        ```
+        (AzureCosmosDBAccount)-[CONTAINS]->(AzureCosmosDBCorsPolicy)
+        ```
+- Azure Database Account contains one or more failover policies.
+
+        ```
+        (AzureCosmosDBAccount)-[CONTAINS]->(AzureCosmosDBAccountFailoverPolicy)
+        ```
+- Azure Database Account is configured with one or more private endpoint connections.
+
+        ```
+        (AzureCosmosDBAccount)-[CONFIGURED_WITH]->(AzureCDBPrivateEndpointConnection)
+        ```
+- Azure Database Account is configured with one or more virtual network rules.
+
+        ```
+        (AzureCosmosDBAccount)-[CONFIGURED_WITH]->(AzureCosmosDBVirtualNetworkRule)
+        ```
+- Azure Database Account contains one or more SQL databases.
+
+        ```
+        (AzureCosmosDBAccount)-[CONTAINS]->(AzureCosmosDBSqlDatabase)
+        ```
+- Azure Database Account contains one or more Cassandra keyspace.
+
+        ```
+        (AzureCosmosDBAccount)-[CONTAINS]->(AzureCosmosDBCassandraKeyspace)
+        ```
+- Azure Database Account contains one or more MongoDB Database.
+
+        ```
+        (AzureCosmosDBAccount)-[CONTAINS]->(AzureCosmosDBMongoDBDatabase)
+        ```
+- Azure Database Account contains one or more table resource.
+
+        ```
+        (AzureCosmosDBAccount)-[CONTAINS]->(AzureCosmosDBTableResource)
+        ```
+
+## AzureCosmosDBLocation
+
+Representation of an Azure CosmosDB Location.
+
+| Field | Description |
+|-------|-------------|
+|firstseen| Timestamp of when a sync job discovered this node|
+|lastupdated| Timestamp of the last time the node was updated|
+|**id**| The unique identifier of the region within the database account.|
+|locationname | The name of the region.|
+|documentendpoint | The connection endpoint for the specific region.|
+|provisioningstate | The status of the Cosmos DB account at the time the operation was called.|
+|failoverpriority | The failover priority of the region.|
+|iszoneredundant | Flag to indicate whether or not this region is an AvailabilityZone region.|
+
+### Relationships
+
+- Azure Database Account has write permissions from, read permissions from and is associated with Azure CosmosDB Locations.
+
+        ```
+        (AzureCosmosDBAccount)-[WRITE_PERMISSIONS_FROM]->(AzureCosmosDBLocation)
```

Comment:
I'm a little confused about the wording of this since I am not familiar with Cosmos. What does it mean for a CosmosDBAccount to have read permissions from a CosmosDBLocation versus having it be associated with a CosmosDBLocation?

Reply:
`read_locations` : An array that contains of the read locations enabled for the Cosmos DB account.

`write_locations`: An array that contains the write location for the Cosmos DB account.

`locations`: This is the one which creates the "associated with" relationship. An array that contains all of the locations enabled for the Cosmos DB account. Also, specifies a region in which the Azure Cosmos DB database account is deployed.

Reply:
Maybe something like `CAN_READ` and `CAN_WRITE`?

Reply:
Database accounts can't read a location. They will have read permission from a location. Likewise for write permissions.

Reply:
What do you mean by "They will have read permission from a location"? For a particular database and location, do you mean the accounts will have permissions to read the data when accessing the database through that location?

How about `CAN_READ_FROM`?

Reply:
Yes, you are right. Okay! I'll change it to CAN_READ_FROM and CAN_WRITE_TO.